LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -328,7 +328,7 @@ DWORD ReadFromPipe(LPVOID Param)
         size_t count = 0;
         size_t lastNewline = 0;
         clearBuffer(chBufOut);
-        while (*ptr > 0 && count < BUFSIZE) {
+        while (*ptr != '\0' && count < BUFSIZE) {

```

Comment:
Any exact reason why *ptr > 0 condition does not suffice? I see it been used in other parts of the code.

Reply:
For multi-byte, mostly they are negative integers, so they will fail the test. The test only works for original ASCII.